client/service.go

```diff
@@ -142,13 +142,25 @@ func (svr *Service) keepControllerWorking() {
 	maxDelayTime := 20 * time.Second
 	delayTime := time.Second
 
+	cutoffTime := time.Now().Add(time.Second * time.Duration(60))
+	reconnectDelay := time.Second
+
 	for {
 		<-svr.ctl.ClosedDoneCh()
 		if atomic.LoadUint32(&svr.exit) != 0 {
 			return
 		}
 
 		for {
+			time.Sleep(reconnectDelay)
```

Comment:
If `reconnectDelay` begins with 1 second, we may delay the normal relogin request. It will reduce the availablity.

We could optimize the reconnectDelay in one minute to `0, 0, 0, 1, 2, 4, 8, 16...`.

Give the first three retry with no delay, so users will not be affected in most scenario.

Reply:
fix